exporter/syslogexporter/README.md

```diff
@@ -0,0 +1,111 @@
+# Syslog Exporter
+
+**Stability level**: Alpha
```

Comment:
We need a Markdown table here that lists the stability, pipeline types and distros like every component has, see example [here](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/3191d9cc96585002e2ab2d6a739c42e3f9e5fe2f/exporter/zipkinexporter/README.md#L3-L7).

We should probably start with [Development](https://github.com/open-telemetry/opentelemetry-collector/blob/main/README.md#development) stability, not Alpha.

Reply:
changed